core/sections/clause_7_core.adoc

```diff
@@ -504,12 +504,151 @@ NOTE: This schema can also be obtained from https://raw.githubusercontent.com/op
+an execute request as show in the following example:
+
+```
+"inputs": {
+   "inputName1": <input value 1>,
+   "inputName2": <input value 2>,
+   etc.
+}
+```
+
+The actual name of each input is its identifier as specified by the input's definition in the <<sc_process_description,process description>>.
+
+====== Cardinality
+
+NOTE: The `maxOccurs` parameter is deprecated in this standard and will be removed in a subsequent version of this document.  See <<deprecated-maxOccurs>> for details.
+
+The cardinality on an input is specified using the `minOccurs` and `maxOccurs` parameters from the input's definition in the <<sc_process_description,process description>>.  The default values of `minOccurs` and `maxOccurs` are 1 indicating that a single input of the corresponding name must be specified in an execute request.  The following table covers the various combinations of `minOccurs` and `maxOccurs` values.
+
+[[cardinality_rules]]
+.Cardinality rules for process inputs.
+[cols="10,10,60,40",options="header"]
+|===
+|minOccurs |maxOccurs |Interpretation |Example
+|0 |0 |Not Allowed |
+|0 |1 |A single input value *may* be optionally specified. | "input": {value}
+|1 |1 |A single input value *must* be specified. | "input": {value}
+|1 |N |At least 1 input value *must* be specified.  +
+Regardless of the number of input values provided, those values *must* be encoded using an array. | "input": [{value}] OR +
+"input": [{value1},...,{valueN}]
+|M |N |All values provided *must* be encoded using an array. |"input": [{value1},...,{valueM}] OR +
+"input": [{value1},...,{valueM},...,{valueN}]
+|===
+
+NOTE: The tokens `{value}`, `{value1}`, `{valueN}` or `{valueM}` represent values of the type expected for the input according to its definition.  This can include array values.
+
+====== Interaction of `minOccurs`/`maxOccurs` and `minItems`/`maxItems`
+
+The `schema` member of the <<process-input-schema,input definition>> in the <<sc_process_description,process description>> defines the schema of a single instance of an input.  If the input happens to be an array then the `minItems` and `maxItems` properties may be used to define the limits of the array.  The use of `minItems` and `maxItems` in the definition of the input does not affect how `minOccurs` and `maxOccurs` are interpreted and the same <<cardinality_rules,cardinality rules>> apply.
+
+Consider the following definitions of an input named "input".
+
+[[schema_examples]]
+.Schema examples for cardinality
+[cols="50a,50a",options="header"]
+|===
+|Example Schema| Examples instances
+|
+[source,yaml]
+----
+inputs:
+  input:
+    schema:
+      type: array
+      maxItems: 2
+      items:
+        type: string
+----
 
-The name of each input is its identifier as specified by the input's definition in the <<sc_process_description,process description>>.
+In this case the schema on an input value is defined as an array and so will always be encoded as an array in an execute request.
```

Comment:
Should cardinality be considered of 1 or 2 here?

If cardinality is 1 (ie: a single input that happens to be a value represented as an array of max 2 items), then how does one differentiate this array from the `imagesInput` example that uses the same representation for cardinality >1.

If cardinality is 2 (ie: 1 or 2 distinct values passed as an array), how should one represent a single "complex" input that is represented as an array allowing at most 2 sub-elements?

Reply:
The images input example has been updated to use minOccurs/maxOccurs.